parsl/providers/azure/template.py

```diff
@@ -0,0 +1,14 @@
+template_string = """#!/bin/bash
+cd ~
```

Comment:
In the AWS version of this, commit 604d610d7d7e5d883c643daeba09f5b65db943ce added in
``
export DEBIAN_FRONTEND=noninteractive
``
before running any `apt` commands, to stop occasional interactive prompts.

See #1094, #1098 

That is probably a useful setting here too.